### DISCUSSION TITLE
UG DQ 03 — Information Artifacts and Aboutness

### DISCUSSION THREAD

**Discussion by johnbeve** [Applied-Ontology-Education/Ontology-and-Intel-Analysis#3] [Undergraduate Discussion Questions]
**GitHub Discussion title:** `UG DQ 03 — Information Artifacts and Aboutness`

**Maximum length:** 500 words

Complete the Week 3 readings and lecture before responding.

## Prompt

Choose one information artifact relevant to intelligence analysis.

Examples include an intelligence report, collection requirement, analyst note, warning, intelligence estimate, source report, credential, message, or briefing.

Explain:

1. What kind of information artifact is it?
2. What is it **about**?
3. Who creates, uses, or receives it?
4. What should an ontology represent about the artifact itself rather than about the thing it describes?
5. What mistake would result from failing to distinguish the artifact from its subject matter?

Include at least one candidate relation or design-pattern statement connecting the artifact to another entity.

**Comment by drewbonge12** [comment]
A credential is an example of an information artifact. A credentials specific use, as mentioned in On Credentials, is defined as a document that is used to achieve a certain end. There are many kinds of credentials, such as a degree, a license, or a passport which all allow our ability to exercise our rights; However, it is important to know that a credential does not grant our rights, it only grants our ability to exercise rights.  
A credential, or specifically a license, is about a particular person. It includes ICEs, or Information Content Entities which are aimed at identifying something in reality. A license may include ICE information such as name, date of birth, sex, height, and eye eolor. A valid credential should also include other information such as the issuer of the credential, the date of issue, and expiration date, as well as an ID number. All of which describe the aboutness of a particular person bearing a license, or information of the credential itself. 
Institutions such as government agencies, educational institutions, and employers may all give out credentials. For example, a driver's license can be issued by the Department of Motor Vehicles, then be used by an individual to exercise their ability to drive a car, as well as serving as a general legal identification credential. The explicit design pattern follows the process where the issuer must issue a credential, and the credential identify a person 
A successful ontology should be able to represent specific information about the credential itself, proving its legitimacy. As previously mentioned, a driver's license displays ICE information such as name, date of birth, sex, height, and eye color; However, that does not describe the credential, it describes a particular individual. Rather than describing a particular individual, a successful ontology should represent metadata of a particular credential. This information could be seen displayed as a date of issue, the expiration date, an ID number, as well as who issued the credential.  
One mistake that may be the result of failing to distinguish a license from a particular person would be allowing an individual with an expired license to operate a motorized vehicle. Yes, the data about that particular person may be correct, but the data about the credential itself is expired, meaning they cannot exercise the ability to drive until they get a new license.

**Comment by LaibahA** [comment]
One kind of information artifact is a map, which is typically a visual information artifact that is used to represent geospatial information. It shows how a certain place or concept relates to information. To clarify, a map isn't about the subject itself, it's an information artifact about the subject. A specific kind of map I will choose to discuss to make this topic a bit narrower, is a Modified Combined Obstacle Overlay (MCOO) as mentioned in the IAO-Intel reading. It represents things like terrain, obstacles for military movement, and other points relevant to military planning. It's about real world geographic information in an area of interest for people involved with military operations. 
A MCOO map would likely be created by a few different kinds of professionals working together, such as intelligence analysts, people involved in the military planning, and geospatial specialists. It would be used by those planning for military operations who need to interpret and analyse the operational area, like military commanders and personell, or other intelligence analysts who need to make reports about the conditions of the area. It could tell them important things like if a plan is unsafe due to the terrain, or if they need prep work because of an obstacle. These are people who can receive the map, groups and organizations involved in the matters could also be the recipients. 
Since the map is about the subject, an ontology should represent information about the map as an information entity, so not necessarily about the subject the map is about. It should essentially represent metadata about the map, so things like what geographical location the map is about, the timeframe it's relevant for, when it was created, who created it, what format it's in (because the maps content can be modeled as an ICE, but it still is generally dependent on an IBE like physical paper or digital picture, according to the reading), etc.
A mistake that could result from failing to distinguish the artifact from its subject matter is taking the map and treating it as if its the explicit information it represents. For example, a map might have shapes on it to represent some sort of pattern (a symbol), and if the artifact isn't properly distinguished from the subject matter, there could be confusion about the map symbol and the actual real world thing. For example, a shaped symbol might represent the optimal location for something, but the shape of the symbol doesn't actually denote the shape of the optimal location. Someone might end up confusing the reality of the content with the representation of the content. One thing an ontology can do to avoid this create distinction by establishing explicit relations, like representing how the symbol is about the obstacle, and clarifying what the "obstacle" might actually be.

**Comment by adriankoz-jos** [comment]
A highly relevant intelligence artifact in the field of intelligence analysis is an intelligence report.

An intelligence report is an Information Content Entity (ICE) that can be produced in one of many ways, but is most often concretized in an Information Bearing Entity (IBE) such as a paper printout or digital file. Intelligence reports are most often created by agencies or a number of analysts working together toward a common goal, are generally produced for the purpose of informing either military or civilian decision-making, and are **about** a specific topic of relevant intelligence: for instance, the FSB, Russia's intelligence agency, may produce and submit to the Duma, a legislative body, an intelligence report about the attritional casualty rate of Russian soldiers (the report being _about_ both the number of casualties _and_ the rate at which they occur) on the front lines in Ukraine, with the goal of informing decision making about related topics such as the rate of conscription or funding for fiber-optic drone countermeasures. 

An ontology should be sure to clearly delineate the report as an ICE, with the associated features of an ICE (containing an IBE, author(s), and so on) and an ontology that conflated the report itself with its subject matter would result in an incomprehensible mess wherein the report would both lose the ability to be indexed for further analysis, and would also somehow appear within the ontology as if the FSB produced a battlefield full of dying soldiers, or else inflicted horrifying casualty rates on the Duma.
